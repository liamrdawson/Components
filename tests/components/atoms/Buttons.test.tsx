import React from 'react'
import { jsx } from '@emotion/react'
import {render} from '@testing-library/react'
import { PrimaryButton, SecondaryButton, TertiaryButton } from '../../../src'
import { defaultTheme } from '../../../src/themes'

test('Snapshot Test: Primary Button', () => {
  const {container} = render(
    <PrimaryButton
      isDisabled={false}
      label="Primary Button"
      theme={defaultTheme}
    />
  )
  expect(container).toMatchSnapshot()
})

test('Snapshot Test: Secondary Button', () => {
  const {container} = render(
    <SecondaryButton
      isDisabled={false}
      label="Secondary Button"
      theme={defaultTheme}
    />,
  )
  expect(container).toMatchSnapshot()
})

test('Snapshot Test: Tertiary Button', () => {
  const {container} = render(
    <TertiaryButton
      isDisabled={false}
      label="Tertiary Button"
      theme={defaultTheme}
    />,
  )
  expect(container).toMatchSnapshot()
})
