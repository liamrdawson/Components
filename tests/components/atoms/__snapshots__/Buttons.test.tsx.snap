// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Test: Primary Button 1`] = `
<div>
  <button
    class="css-1195o8y-PrimaryButton"
    type="button"
  >
    Primary Button
  </button>
</div>
`;

exports[`Snapshot Test: Secondary Button 1`] = `
<div>
  <button
    class="css-1ou42su-SecondaryButton"
    type="button"
  >
    Secondary Button
  </button>
</div>
`;

exports[`Snapshot Test: Tertiary Button 1`] = `
<div>
  <button
    class="css-11f3soy-TertiaryButton"
    type="button"
  >
    Tertiary Button
  </button>
</div>
`;
